libraries/ui-shared/Dialog/components/PipelineErrorDialog/index.jsx

```diff
@@ -99,32 +100,65 @@ class PipelineErrorDialog extends Component {
 
   /**
    * Renders the error message in developer mode.
-   * @param {Object} params The message parameters.
    * @returns {JSX}
    */
-  renderDevErrorMessage = params => (
-    <div aria-hidden>
-      <strong>Pipeline:</strong> {params.pipeline}<br />
-      { params.code && (
-        <span>
-          <strong>Code: </strong> {params.code} <br />
-        </span>
-      )}
-      <strong>Message:</strong> {params.message} <br />
-      <p>
-        <strong>Params:</strong><br />
-        {JSON.stringify(params.request, null, ' ')}
-      </p>
-    </div>
-  );
+  renderDevErrorMessage = () => {
+    const { params } = this.props;
+
+    return (
+      <div aria-hidden>
+        <strong>Pipeline:</strong> {params.pipeline}<br />
+        { params.entityId && (
+          <span>
+            <strong>Entity id: </strong> {params.entityId} <br />
+          </span>
+        )}
+        { params.code && (
+          <span>
+            <strong>Code: </strong> {params.code} <br />
+          </span>
+        )}
+        <strong>Message:</strong> {params.message} <br />
+        { params.messageParams && (
+          <p>
+            <strong>Message Params:</strong><br />
+            {JSON.stringify(params.messageParams, null, ' ')}
+          </p>
+        )}
+        { params.translated && (
+          <span>
+            <strong>Message Translated: </strong> {params.translated.toString()} <br />
+          </span>
+        )}
+        <p>
+          <strong>Request Params:</strong><br />
+          {JSON.stringify(params.request, null, ' ')}
+        </p>
+      </div>
+    );
+  }
 
   /**
    * Renders the regular error message in user mode.
    * @returns {JSX}
    */
-  renderUserErrorMessage = () => (
-    <I18n.Text string={this.props.message || 'modal.body_error'} />
-  );
+  renderUserErrorMessage = () => {
+    const { message = '', params = {} } = this.props;
+
+    return (
+      <React.Fragment>
+        {params.translated === true && (
```

Comment:
Is this really meant to be here? In the worst case, this would output `modal.body_error` as plain text.

Reply:
You are right, `modal.body_error` was meant supposed to be the param for a `I18n.Text` Component. I will correct that.